packages/supabase_flutter/lib/src/supabase.dart

```diff
@@ -75,6 +75,7 @@ class Supabase {
     PostgrestClientOptions postgrestOptions = const PostgrestClientOptions(),
     StorageClientOptions storageOptions = const StorageClientOptions(),
     FlutterAuthClientOptions authOptions = const FlutterAuthClientOptions(),
+    Future<String> Function()? accessToken,
```

Comment:
Maybe create a typedef for this?

Reply:
Thanks for the feedback! I know this is highly opinionated, but I really feel that using typedef generally decreases the readability of the code. Especially since a property named `accessToken` doesn't sound like a function.